src/Microsoft.AspNetCore.Razor/Editor/SpanEditHandler.cs

```diff
@@ -29,8 +29,9 @@ public SpanEditHandler(Func<string, IEnumerable<ISymbol>> tokenizer, AcceptedCha
         public AcceptedCharacters AcceptedCharacters { get; set; }
 
         /// <summary>
-        /// Provides a set of hints to editors which may be manipulating the document in which this span is located.
+        /// This property is obsolete and will be removed in a future version.
```

Comment:
I think this is a sufficiently obscure part of the system that this comment is just fine.